packages/webpack/src/runtime/runtime.js

```diff
@@ -247,8 +247,8 @@ const lavamoatRuntimeWrapper = (resourceId, runtimeKit) => {
     // TODO: print a warning for other functions on the __webpack_require__ namespace that we're not supporting.
     //   It's probably best served at build time though - with runtimeRequirements or looking at the items in webpack runtime when adding lavamoat runtime.
 
-    // The following seem harmless and are used by default: ['O', 'n', 'd', 'o', 'r', 's', 't']
-    const supportedRuntimeItems = ['O', 'n', 'd', 'o', 'r', 's', 't']
+    // The following seem harmless and are used by default: ['O', 'n', 'd', 'o', 'r', 's', 't', 'b']
+    const supportedRuntimeItems = ['O', 'n', 'd', 'o', 'r', 's', 't', 'b']
```

Comment:
what's b?

Reply:
@weizman you know the rest of them? 🤣

Reply:
It's where webpack keeps its basepath to figure out how dynamic requires fit into the bundle.
Fragment from runtime:
```
__webpack_require__.b = document.baseURI || self.location.href;.
```